index.js

```diff
@@ -26,6 +26,8 @@ const ServerResponse = require('fwsp-server-response');
 let serverResponse = new ServerResponse();
 const UMFMessage = require('fwsp-umf-message');
 
+const RedisConnection = require('./redis');
```

Comment:
I'd imagine for something more significant than changing the `maxReconnectionAttempts` or `maxDelayBetweenReconnections` options, it'd be best to extend the RedisConnection class.